CuratedContent/WhatIsDevOps.md

```diff
@@ -0,0 +1,31 @@
+# What is DevOps?
+
+## Mueller, Ernest, 2010, https://theagileadmin.com/what-is-devops
+
+DevOps has become a catch-all term in agile organizations, but defiing what it *is* turns out not to be simple.  This blog post attempts to sort through the various ideas surrounding DevOps and bring some clarity to what is, at heart, a large and nuanced concept.
+
+> DevOps means a lot of different things to different people because the discussion around it covers a lot of ground.
+
+Mueller does a nice job of discussing many of the complexities and trade-offs in all this ground including a bit of history lesson, but two items stood out the most to me.
+
+1. The discussion focuses on web apps - a common issue for HPC practioners. Don't let that stop you from understanding and applying these concepts.
```

Comment:
I *love* the use of *practitioners* here ;)

Hmm...what is a common issue thought...that most DevOps literature is steeped in the web app domain or that web apps are a domain that HPC practitioners commonly struggle with?

Maybe revise to "Although the discussion focuses on web apps, HPC practitioners will still benefit from many of the concepts."

Reply:
My thought was that most DevOps literature is focused on web applications not that HPC people had understanding. I like your re-write, thanks!